plugins/permission-common/report.api.md

```diff
@@ -183,9 +183,16 @@ export class PermissionClient implements PermissionEvaluator {
   ): Promise<QueryPermissionResponse[]>;
 }
 
+// @public
+export type PermissionClientOptions = {
+  discovery: DiscoveryApi;
+  config: Config;
+};
+
 // @public
 export type PermissionClientRequestOptions = {
   token?: string;
+  permissionIdentity?: PermissionIdentity;
```

Comment:
After https://github.com/backstage/backstage/pull/30127 we have more deliberately well defined `toString` and `toJSON` forms of backend identities. Would it make sense to make this just a string, and calling it something like `identityKey` or so? Then typical backend callers can pass in `String(credentials)`

Reply:
Done